setup.cfg

```diff
@@ -12,3 +12,11 @@ exclude = venv,build
 [yapf]
 based_on_style = pep8
 column_limit = 99
+
+# This section is for mypy.
+[mypy]
+disallow_untyped_defs = True
+ignore_missing_imports = True
+
+[mypy-optuna.storages.redis_storage]
+strict_optional = False
```

Comment:
strict_optional = False made the pull request pass the tests, otherwise it complains about 

Argument 1 to "loads" has incompatible type "Optional[bytes]"; expected "bytes"

in all of redis get statements.
Perhaps you can cast some light on that error which I do not have much experience with.

Reply:
Sorry for my earlier brief comment. I think the errors derive from `pickle.loads` and `float`? mypy complains when you call  `pickle.loads` with arguments that may be `None` (type hinted to be `Optional`).  `pickle.loads` apparently does not expect that. Similarly for `float`. What you can do is to create temporaries and use `assert`s or explicitly check the type using e.g. `if`s, to let mypy know that these values cannot be `None`. 

**Current**
```python
pickle.loads(self.redis.get(foo))  # `self.redis.get` might return `None`?
```
**Possible solution**
```python
bar = self.redis.get(foo)
assert bar is not None
# or... 
# if bar is None:
#     ....  # e.g. raise an error here or call `pickle.loads` in the `else` clause.
pickle.loads(bar)  
```

Reply:
You can run `mypy` locally, e.g. 

```bash
mypy optuna/storages/redis_storage.py
```

Reply:
Thanks..addressed all Optional[bytes] coming from redis